apps/console/src/features/identity-providers/components/settings/attribute-management/attribute-selection-v2.tsx

```diff
@@ -0,0 +1,214 @@
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { Field, Form } from "@wso2is/form";
+import { EmptyPlaceholder, Heading, Hint, LinkButton, PrimaryButton } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Divider, Grid, Icon, Segment } from "semantic-ui-react";
+import { AddAttributeSelectionModal } from "./attribute-selection-modal";
+import { AttributeMappingList } from "./attributes-mapping-list";
+import { getEmptyPlaceholderIllustrations } from "../../../../core";
+import { IdentityProviderClaimInterface, IdentityProviderCommonClaimMappingInterface } from "../../../models";
+
+export interface AttributesSelectionV2Props extends TestableComponentInterface {
+    attributeList: Array<IdentityProviderClaimInterface>;
+    mappedAttributesList: Array<IdentityProviderCommonClaimMappingInterface>;
+    onAttributesSelected: (mappingsToBeAdded: IdentityProviderCommonClaimMappingInterface[]) => void;
+}
+
+export const AttributesSelectionV2: FunctionComponent<AttributesSelectionV2Props> = (
```

Comment:
Add a doc comment for component.

Reply:
Addressed this in commit b49a285. Adde doc comments for PR components.